custom/yeksi_naa_reports/filters.py

```diff
@@ -43,13 +43,14 @@ class MonthsDateFilter(BaseReportFilter):
 
     @classmethod
     def months(cls):
-        month_pairs = []
-        for month_number in range(1, 13):
-            month_pairs.append({
-                'name': calendar.month_name[month_number],
-                'value': month_number,
-            })
-        return month_pairs
+        return [
+            {u'name': 'Janvier', u'value': 1}, {u'name': 'Février', u'value': 2},
```

Comment:
As a heads up, the ```u``` prefixes in front of unicode string literals are unnecessary because this file contains the import ```from __future__ import unicode_literals```.